### PR TITLE
Enable dual code hash by disabling constraints

### PR DESCRIPTION
**Pull request by z2trillion**

### CODE REVIEW

**Comment by lispc** [comment]
@noel2004 let us check whether this branch can be used in https://github.com/scroll-tech/scroll-zkevm/pull/106 to enable zktrie feature, or we have to use https://github.com/scroll-tech/zkevm-circuits/pull/309 alongside this branch

File: src/serde.rs [hunk 1]
```diff
@@ -276,13 +276,23 @@ pub struct AccountData {
     /// balance
     #[serde(deserialize_with = "de_uint_hex", serialize_with = "se_uint_hex")]
     pub balance: BigUint,
-    /// codeHash
+    /// keccakCodeHash
```
Comment (noel2004) [inline src/serde.rs:279]: still I prefer naming it as `codeHash` here so we can kept compatible on code with zkevm-circuit

**Comment by noel2004** [comment]
LGTM with trivial renaming

File: src/serde.rs (hunk 1 above)
Comment (z2trillion) [reply src/serde.rs]: we were actually planning to rename it to `keccakCodeHash` in https://github.com/scroll-tech/zkevm-circuits/pull/309, but I think we can undo it there too.

**Pull request merged**